src/Symfony/Component/Serializer/Serializer.php

```diff
@@ -51,7 +51,7 @@ class Serializer implements SerializerInterface, ContextAwareNormalizerInterface
      * Flag to control whether an empty array should be transformed to an
      * object (in JSON: {}) or to a map (in JSON: []).
      */
-    public const EMPTY_ARRAYS_AS_OBJECT = 'empty_iterable_as_object';
+    public const EMPTY_ARRAYS_AS_OBJECT = 'empty_arrays_as_object';
```

Comment:
I agree with this. 
This constant has not been released yet so it is not a BC break.